src/Groups/matrices/form_group.jl

```diff
@@ -928,18 +929,20 @@ function _isometry_group_via_decomposition(
   depth::Int=-1,
   bacher_depth::Int=0,
   use_heuristics::Bool=false,
-  closed::Bool=true,
   direct::Bool=true,
-  set_nice_mono::Bool=true,
+  set_nice_mono::Bool=true
```

Comment:
Why do you remove the commas ? They don't break anything, and I believe it is actually suggested to do so. In particular, it helps adding new arguments without forgetting a comma (at least that's a way I interpret it)